BearlyHumans/squirrelville
# Finished adding events + SFX and Particles controllers

- Squirrel Controller now has SFX events (using the same triggers as Animation Events)
- Also has Particles Events
- All three event lists now have descriptive element names
- SFX controller has been created and should be good for most purposes
- SFX controller pauses/resumes sounds when the game is paused
- Particles controller created and can play and stop particles
- Animation events are fixed on a new Player Package Prefab (in case it would cause conflicts)

Test:
* Go on Jake Island OR use the Player Package W/ Controllers OR fix all the animation events manually
* Check what SFX events are listed, and if they match what plays when you trigger the event
          ** Prefab has Ball Descent on jump
* Test the different action types for events
* Do crazy stuff like play 20 sounds at once
* Add a particle to the controller and an event to the manager

Notes/Bugs:
> Changes are likely to cause prefab inconsistencies so I made a new one
> Animation events will be reset because of the changes
> Particles Controller is very barebones, probably needs improving
